src/natcap/invest/forest_carbon_edge_effect.py

```diff
@@ -42,18 +42,19 @@
         "results_suffix": spec_utils.SUFFIX,
         "n_workers": spec_utils.N_WORKERS,
         "n_nearest_model_points": {
-            "expression": "value > 0",
+            "expression": "value > 0 and value.is_integer()",
             "type": "number",
```

Comment:
Oh yeah! No idea why I added that 🤔

Reply:
But we still need to change it to `"type": "integer"`, right?

Reply:
Oop you're right!

Reply:
Actually, I was wrong about this. It was correct before because only the `number` type can have an `expression` to limit its values. We should probably consider allowing an `expression` on all the numeric types (number, ratio, percent, integer). But that would be a larger task.

Reply:
My bad!